tasks/kernel_matrix_testing/vmconfig.py

```diff
@@ -55,80 +59,42 @@
     "4.20",
 ]
 distributions = {
-    "ubuntu_18": "bionic",
-    "ubuntu_20": "focal",
-    "ubuntu_22": "jammy",
-    "jammy": "jammy",
-    "focal": "focal",
-    "bionic": "bionic",
+    # Ubuntu mappings
+    "ubuntu_16": "ubuntu_16.04",
```

Comment:
This maps the various distro names to a normalized version. For example both `ubuntu_22` and `jammy` map to `ubuntu_22.04`.
This is added as a convenience so that the user does not have to remember the exact name the scripts require.
Moreover, the name is fuzzy matched to the closest available key so even if the user provides `ubuntu22` as input it will get correctly mapped.